src/components/reset-password-2fa-sms/reset-password-2fa-sms-controller.ts

```diff
@@ -33,6 +33,20 @@ export function resetPassword2FASmsGet(
   return async function (req: Request, res: Response) {
     const { email } = req.session.user;
     const { sessionId, clientSessionId, persistentSessionId } = res.locals;
+
+    if (
+      req.session.user.wrongCodeEnteredLock &&
+      new Date().getTime() <
+        new Date(req.session.user.wrongCodeEnteredLock).getTime()
+    ) {
+      return res.render(
+        "security-code-error/index-security-code-entered-exceeded.njk",
+        {
+          newCodeLink: PATH_NAMES.RESET_PASSWORD_2FA_SMS,
+        }
+      );
+    }
+
```

Comment:
Worth adding some unit tests to verify the screen is rendered if the lock is active? I've done something similar [here](https://github.com/govuk-one-login/authentication-frontend/pull/1363/files#diff-c860c67cf113b4abd7406340bf84427f2f86d34408350ddc156baee241072c59)

Reply:
Will do, thanks Ayo!